packages/cardano-services/package.json

```diff
@@ -29,7 +29,7 @@
     "cleanup:nm": "rm -rf node_modules",
     "cleanup": "rm -rf dist node_modules",
     "cli": "ts-node --transpile-only src/cli.ts",
-    "compose:up": "docker compose --env-file environments/.env.$NETWORK -p cardano-services-$NETWORK -f docker-compose.yml -f ../../compose/common.yml -f ../../compose/pg-agent.yml -f ../../compose/$(uname -m).yml ${FILES:-} up",
+    "compose:up": "__FIX_UMASK__=$(chmod -R a+r ../../compose/placeholder-secrets) docker compose --env-file environments/.env.$NETWORK -p cardano-services-$NETWORK -f docker-compose.yml -f ../../compose/common.yml -f ../../compose/pg-agent.yml -f ../../compose/$(uname -m).yml ${FILES:-} up",
```

Comment:
@mirceahasegan it's not really used as an env var, but I added it as the least invasive way to compose the commands with `chmod -R a+r ../../compose/placeholder-secrets`, which is needed on systems setting `umask 0077`. Otherwise, Postgres (which is not run as `root` inside its container) cannot read the secrets.